web-registry/src/components/molecules/LineItemLabelAbove.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Box from '@mui/material/Box';
+
+import Description from 'web-components/lib/components/description';
+import { Label } from 'web-components/lib/components/label';
+
+interface LineItemProps {
+  label: string;
+  data: string | JSX.Element | number;
+}
+
+export const LineItemLabelAbove: React.FC<LineItemProps> = ({
```

Comment:
looks like we might have some duplicate styles between this PR and my two open PRs. I don't imagine it'll be a big issue, but could be worth circling back on when we merge (depending on which gets merged first)